dokka-subprojects/plugin-base/src/main/resources/dokka/templates/includes/footer.ftl

```diff
@@ -1,7 +1,13 @@
 <#macro display>
-      <div class="footer">
-        <span class="go-to-top-icon"><a href="#content" id="go-to-top-link"></a></span><span>${footerMessage}</span><span
-                class="pull-right"><span>Generated by </span><a
-                href="https://github.com/Kotlin/dokka"><span>dokka</span><span class="padded-icon"></span></a></span>
-      </div>
+    <div class="footer">
+        <a href="#content" id="go-to-top-link" class="footer--button footer--button_go-to-top"></a>
+        <span>${footerMessage}</span>
+        <span class="pull-right">
+            <span>Generated by </span>
+            <a class="footer--link" href="https://github.com/Kotlin/dokka">
+                <span>dokka</span>
+                <span class="footer--external-link-icon"></span>
```

Comment:
If we change node for me `::after` looks better.

Reply:
Definitely! changed this